test/OpenTelemetry.Tests/Metrics/MetricAPITest.cs

```diff
@@ -0,0 +1,57 @@
+// <copyright file="MetricAPITest.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using Xunit;
+
+#nullable enable
+
+namespace OpenTelemetry.Metrics.Tests
+{
+    public class MetricApiTest
+    {
+        [Fact]
+        public void SimpleTest()
+        {
+            using var provider = new MetricProviderBuilderSdk()
```

Comment:
Is this intended to be called `MetricProviderBuilderSdk` or `MeterProviderBuilderSdk`?

Reply:
It should be MeterProvider.